test/cli/test-mutating-admission-policy/check-labels/resource2.yaml

```diff
@@ -0,0 +1,12 @@
+# file: pod-skip.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: skip-pod
+  labels:
+    app: custom
```

Comment:
Is this a skip? We used to evaluate `preconditions` and report a skip if the condition is not met, or skip for anchors in a validate rule.

We could treat this  `!has(object.metadata.labels.app) ?`as the same conditional check and re-use the skip strategy. We need to  document it clearly.